CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 * [BUGFIX] TSDB: Fixed querying ingesters in `LEAVING` state with the experimental TSDB blocks storage. #1854
 * [BUGFIX] TSDB: Fixed error handling in the series to chunks conversion with the experimental TSDB blocks storage. #1837
 * [BUGFIX] TSDB: Fixed TSDB creation conflict with blocks transfer in a `JOINING` ingester with the experimental TSDB blocks storage. #1818
+* [BUGFIX] TSDB: ShipInterval of <=0 treated as disabled instead of allowing panic. #1975
```

Comment:
Being `ShipInterval` an internal naming, I would suggest to replace it with the related CLI flag `-experimental.tsdb.ship-interval`

Reply:
@thorfour The PR has been merged before changing this. Would you mind submitting another PR with this change, just for the sake of clarity, please? /cc @gouthamve

Reply:
@pracucci follow on PR is here: https://github.com/cortexproject/cortex/pull/1979